.changeset/clever-buttons-clean.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/starlight": patch
+---
+
+feat: add i18n nl translations
```

Comment:
We don’t need the “feat” here because of how Changesets will get formatted for the [CHANGELOG](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md) under a heading. For example they end up something like this:

> https://github.com/withastro/starlight/pull/257 [`0502327`](https://github.com/withastro/starlight/commit/050232770c8a2d22f317def8e734e4abb36387c6) Thanks [@JosefJezek](https://github.com/JosefJezek)! - Add Czech language support

So a simple full English sentence is often a good choice, e.g.

```suggestion
Add Dutch UI translations
```